include/smack/SmackRep.h

```diff
@@ -133,6 +166,9 @@ class SmackRep {
   const Expr* cast(const llvm::Instruction* I);
   const Expr* cast(const llvm::ConstantExpr* CE);
   const Expr* cast(unsigned opcode, const llvm::Value* v, const llvm::Type* t);
+  string bopName(unsigned operand1, unsigned operand2, const string& operation);
```

Comment:
Cleaner solution: a variable argument method `opName(const string& op, …)` which returns `op` unless bitvectors are being used, in which case it returns `op + ".iX.iY….iZ"` where `X`, `Y` …, `Z` are the `unsigned` arguments to `opName`.